src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -81,31 +84,35 @@ public static Sentiment getSentiment(String projectID, String lyrics) throws Ill
   }
 
   /**
-   * From every entity generated by the API we get only part of the information (name and salience)
-   * and store the new SongEntities into a new simplified list. 
+   * From every entity generated by the API we get only part of the information 
+   * (name, salience, type and wikiLink) and store the new SongEntities into a new simplified list. 
+   * 
+   * We round the salience value to allow us to actually test these methods.
+   * In the beginning, every entity has only one type, so we just populate the set
+   * with only one string.
    */
   public static List<SongEntity> getSimplifiedEntityList(List<Entity> entityList) {
     List<SongEntity> simplifiedEntityList = new ArrayList<>();
 
     for (Entity entity : entityList) {
       // Use round() here to set the double to 2 decimals.
       SongEntity simplifiedEntity = new SongEntity(entity.getName(), 
-          Math.round(entity.getSalience() * 100.0) / 100.0, entity.getType().toString(),
+          entity.getSalience(), 
+          Sets.newHashSet(entity.getType().toString()),
           entity.getMetadataMap().getOrDefault("wikipedia_url", ""));
       simplifiedEntityList.add(simplifiedEntity);
     }
 
     return simplifiedEntityList;
   }
 
-  public static List<SongEntity> getTopSalientEntities(List<SongEntity> simplifiedEntityList) {
-    List<SongEntity> tempList = new ArrayList<>(simplifiedEntityList);
-    tempList.sort(SongEntity.ORDER_BY_SALIENCE_DESCENDING);
+  public static List<SongEntity> getTopSalientEntities(List<SongEntity> filteredList) {
```

Comment:
Please add a documentation to this method that you modify the list passed as parameter. That might not be obvious to the caller.

Reply:
Since we merged the 2 methods, this is not needed anymore right?

Reply:
Yep. If you accepted the suggestion below, then this method is irrelevant. Make sure to document the other one if what it does is not obvious from the name.